Gems/EMotionFX/Code/EMotionFX/Pipeline/SceneAPIExt/Rules/RootMotionExtractionRule.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/std/smart_ptr/make_shared.h>
+#include <AzCore/Serialization/SerializeContext.h>
+#include <SceneAPI/SceneCore/DataTypes/Groups/IGroup.h>
+#include <SceneAPI/SceneCore/Events/ManifestMetaInfoBus.h>
+#include <SceneAPI/SceneCore/DataTypes/GraphData/IBoneData.h>
+#include <SceneAPIExt/Rules/RootMotionExtractionRule.h>
+
+namespace EMotionFX::Pipeline::Rule
+{
+    void RootMotionExtractionData::Reflect(AZ::ReflectContext* context)
+    {
+        AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
+        if (!serializeContext)
+        {
+            return;
+        }
+
+        serializeContext->Class<RootMotionExtractionData>()
+            ->Version(1)
+            ->Field("sampleJoint", &RootMotionExtractionData::m_sampleJoint)
+            ->Field("transitionZeroX", &RootMotionExtractionData::m_transitionZeroXAxis)
+            ->Field("transitionZeroY", &RootMotionExtractionData::m_transitionZeroYAxis)
+            ;
+
+        AZ::EditContext* editContext = serializeContext->GetEditContext();
+        if (editContext)
+        {
+            editContext->Class<RootMotionExtractionData>("Root motion extraction data", "Root motion extraction data.")
+                ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                    ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                ->DataElement("NodeListSelection", &RootMotionExtractionData::m_sampleJoint, "Sample joint", "Sample joint to extract motion data from. Usually the hip joint.")
+                    ->Attribute("ClassTypeIdFilter", AZ::SceneAPI::DataTypes::IBoneData::TYPEINFO_Uuid())
+                ->ClassElement(AZ::Edit::ClassElements::Group, "Transition Extraction")
+                ->DataElement(AZ::Edit::UIHandlers::Default, &RootMotionExtractionData::m_transitionZeroXAxis, "Force X Axis Transition to 0", "Force X Axis movement to be zero.")
```

Comment:
What about "Ignore X-Axis" or "Ignore X-Axis Movement" here instead? Might be more easy to understand for users?